supabase/migrations/20241118122106_add-updated-dat-on-topics.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE topics
+    ADD COLUMN updated_at timestamptz;
+
+UPDATE
+    topics
+SET
+    updated_at =(
+        SELECT
+            CASE WHEN updated_at IS NOT NULL THEN
+                updated_at
+            ELSE
+                created_at
+            END
+        FROM
+            messages
+        WHERE
+            topics.last_message_id = messages.id);
```

Comment:
Perhaps you could make `updated_at` not nullable after they've all been updated to have an `updated_at` value?

Reply:
It can happen the topic to not have messages at all and in that case the updated_at will be null.